packages/shared-data/json/supported-models.json

```diff
@@ -1,27 +1,106 @@
 [
   {
-    "name": "meta-llama/llama-4-scout-17b-16e-instruct",
-    "displayName": "LLaMA 4 Scout",
-    "rateLimit": 40000000
+    "name": "openai/gpt-oss-120b",
+    "displayName": "OpenAI GPT OSS 120B",
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "bedrock": {
+          "id": "openai.gpt-oss-120b-1:0"
+        }
+      }
+    ],
+    "modality": "chat"
   },
   {
-    "name": "claude-3-5-sonnet-20241022",
-    "displayName": "Claude 3.5 Sonnet",
-    "rateLimit": 0
+    "name": "openai/gpt-oss-20b",
+    "displayName": "OpenAI GPT OSS 20B",
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "bedrock": {
+          "id": "openai.gpt-oss-20b-1:0"
+        }
+      }
+    ],
+    "modality": "chat"
   },
   {
-    "name": "claude-3-haiku-20240307",
-    "displayName": "Claude 3 Haiku",
-    "rateLimit": 0
+    "name": "google/gemini-2.5-flash",
+    "displayName": "Gemini 2.5 Flash",
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "vertex": {
+          "id": "gemini-2.5-flash"
+        }
+      }
+    ],
+    "modality": "chat"
   },
   {
-    "name": "voyage-multimodal-3",
-    "displayName": "Voyage Multimodal 3",
-    "rateLimit": 0
+    "name": "google/gemini-2.5-flash-lite",
+    "displayName": "Gemini 2.5 Flash Lite",
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "vertex": {
+          "id": "gemini-2.5-flash-lite"
+        }
+      }
+    ],
+    "modality": "chat"
   },
   {
-    "name": "voyage-3-large",
+    "name": "anthropic/claude-sonnet-4-20250514-v1",
+    "displayName": "Anthropic Claude Sonnet 4 (20250514)",
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "bedrock": {
+          "id": "anthropic.claude-sonnet-4-20250514-v1:0"
+        }
+      }
+    ],
+    "modality": "chat"
+  },
+  {
+    "name": "anthropic/claude-3-haiku-20240307-v1",
+    "displayName": "Anthropic Claude Haiku 3 (20240307)",
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "bedrock": {
+          "id": "anthropic.claude-3-haiku-20240307-v1:0"
+        }
+      }
+    ],
+    "modality": "chat"
+  },
+  {
+    "name": "voyage/voyage-3-large",
     "displayName": "Voyage 3 Large",
-    "rateLimit": 400000000
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "voyage": {
+          "id": "voyage-3-large"
+        }
+      }
+    ],
+    "modality": "embedding"
+  },
+  {
+    "name": "amazon/titan-embed-text-v2",
```

Comment:
Is titan strong enough for us to be used as default or do we need another model?